dist/t/spec/spec_helper.rb

```diff
@@ -39,7 +39,7 @@ def login
     visit "/user/login"
     fill_in 'user_login', with: 'Admin'
     fill_in 'user_password', with: 'opensuse'
-    click_button('log-in-button')
+    click_button 'Log In »'
```

Comment:
Did you try it?

https://github.com/openSUSE/open-build-service/blob/master/src/api/spec/support/features/features_authentification.rb#L7

We do the same since one year in our rspec testsuite ...